docs/source/spark_overview.rst

```diff
@@ -288,74 +288,34 @@ This section describes how you can run Spark jobs on any number of HPC compute n
 The scripts and examples described here rely on the SLURM scheduling system and have been tested
 on NREL's Kestrel cluster.
 
-NREL's HPC GitHub [repository](https://github.com/NREL/HPC) contains scripts that will create an
-ephemeral Spark cluster on compute nodes that you allocate.
-
-The [README](https://github.com/NREL/HPC/blob/master/applications/spark/README.md) in the
-repository has generic instructions to run Spark in a variety of ways. The rest of this section
-calls out choices that you should make to run Spark jobs with dsgrid.
-
-.. note:: The latest scripts currently supporting Kestrel are at this branch:
-   https://github.com/daniel-thom/HPC/tree/kestrel-update. Please follow its README instead.
-
-1. Clone the repository.
-
-.. code-block:: console
-
-    $ git clone https://github.com/NREL/HPC.git
-
-2. Choose compute node(s) with fast local storage. This example will allocate one node.
+Install the Python package ``sparkctl`` to run the scripts described below. You can install it with
+this command:
 
 .. code-block:: console
 
-    $ salloc -t 01:00:00 -N1 --account=dsgrid --partition=debug --tmp=1600G --mem=240G
+    $ pip install "sparkctl[pyspark]"
 
-- NREL's Kestrel cluster will let you allocate one debug job each with two nodes. So, you can use
-  these scripts to create a two-node cluster for one hour.
-- If the debug partition is not too full, you can append ``--qos=standby`` to the command above
-  and not be charged any AUs.
+The [sparkctl documentation](https://nrel.github.io/sparkctl/) has generic instructions to run
+Spark in a variety of ways. The rest of this section calls out choices that you should make to run
+Spark jobs with dsgrid.
 
-3. Select a Spark container compatible with dsgrid, which currently requires Spark v3.5.2.
-   The team has validated the container ``/datasets/images/apache_spark/spark352_py311.sif``. It
-   was created with this Dockerfile in dsgrid: ``docker/spark/Dockerfile``. The container includes
-   ipython, jupyter, pyspark, pandas, duckdb, and pyarrow, but not dsgrid. The
-   ``configure_and_start_spark.sh`` will normally be updated to use the currently-supported dsgrid
-   container by default, but there may be some cases where you need to specify it manually with the
-   ``-c`` option.
+1. Choose compute node(s) with fast local storage. This example will allocate one node.
```

Comment:
I wonder if you should link documentation on HPC nodes, maybe this? https://nrel.github.io/HPC/Documentation/Systems/Kestrel/Filesystems/#node-file-system